### ISSUE TITLE
Can CascadeForestClassifier.predict() return the original class labels instead of 0/1?

### ISSUE THREAD

**Issue by NiMaZi** [LAMDA-NJU/Deep-Forest#13]
Trying the basic tutorial:

```
X, y = load_digits(return_X_y=True)
y += 100 # This is what I changed
X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=1)

model = CascadeForestClassifier(random_state=1)

# Train
model.fit(X_train, y_train)

# Evaluate
y_pred = model.predict(X_test)
acc = accuracy_score(y_test, y_pred) * 100

print("Testing Accuracy: {:.3f} %".format(acc))
```
And get:
```
Testing Accuracy: 0.000 %
```

This is because the class labels are made into "101", "102", "103", ..., instead of "1", "2", "3",... .
But the predict() function (or the model itself) could not deal with these labels.

Is it possible to let the CascadeForestClassifier.predict() function use the original class label (e.g., "101" instead of "1")? It is the basic feature of sklearn models. Or it is also fine to explain in the documentation how CascadeForestClassifier maps original class labels into integers. Now it is a bit confusing and not very convenient to use.

BTW, big fan of your work! :))) I've been waiting for it even since it is published on IJCAI.

**Comment by xuyxu** [comment]
Thanks for your issue and suggestions! I agree with you that it would be better if `CascadeForestClassifier` can support an internal label encoder. I will add this feature request in the pinned issue, and are you willing to work on this? Help from the community is always welcomed ;-)

**Comment by xuyxu** [comment]
I will work on this feature request ;-)

**Comment by xuyxu** [comment]
Hi @NiMaZi, are you working on this feature request? I am much willing to leave this feature request to you, I can then work on other parts ;-)

**Comment by NiMaZi** [comment]
Yes, I am working on it.

Yi-Xuan Xu <notifications@github.com> 于2021年2月3日周三 下午5:04写道：

**Comment by xuyxu** [comment]
Closed via PR #18.
